specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-01-01/Marketplace.json

```diff
@@ -0,0 +1,451 @@
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreProperties"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetPrivateStoreByName": {
+            "$ref": "./examples/GetPrivateStoreByName.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes the private store. All that is not saved will be lost.",
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded."
+          },
+          "404": {
+            "description": "Could not find the store"
+          }
+        }
+      },
+      "put": {
+        "description": "Changes private store properties",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "payload",
+            "schema": {
+              "$ref": "#/definitions/PrivateStoreProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Change successful"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Marketplace/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists all of the available Microsoft.Marketplace REST API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Microsoft.Marketplace error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "PrivateStoreIdParameter": {
+      "name": "PrivateStoreId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The store ID - must use the tenant ID"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the request."
+    },
+    "OfferIdPathParameter": {
+      "in": "path",
+      "name": "OfferId",
+      "type": "string",
+      "required": true,
+      "description": "The offer ID to update or delete"
+    },
+    "offerObjectParameter": {
+      "in": "body",
+      "name": "offerObject",
+      "schema": {
+        "$ref": "#/definitions/offerSkuObject"
+      }
+    },
+    "offerSkuObjectParameter": {
```

Comment:
>offerSkuObjectParameter [](start = 5, length = 23)

offerSkuObjectParameter is unsed. remove.

Reply:
was removed